src/deploy/db/mongo-adapter/mongo-adapter.ts

```diff
@@ -84,7 +86,9 @@ export class MongoDBAdapter {
 
   // Contract methods
   async getContract (contractName : string, version ?: string) {
-    if (!version) version = await this.getCheckLatestVersion();
+    if (!version) {
+      ({ dbVersion: version } = await this.getCheckLatestVersion());
```

Comment:
out parens dont seem necessary `( ... )`

Reply:
since it's not const, they are. it shows an error otherwise